src/components/pages/ParentDashboard/ParentDashboardContainer.js

```diff
@@ -41,6 +42,7 @@ function ParentDashboardContainer({ LoadingComponent, ...props }) {
           authService={authService}
         />
       )}
+      <Footer />
```

Comment:
Any particular reason for integrating the Footer on the Redux Container component rather than the Render component?